### DISCUSSION TITLE
Possibility of including .md pages outside parent repository?

### DISCUSSION THREAD

**Discussion by beyond9thousand** [retypeapp/retype#451] [Q&A]
Would it be possible to include markdown files from outside the repository to be deployed on the website as a page? I know that I can just copy the file, but I was looking for a solution where I could somehow either link to the page in my repository and that would detect the content appropriately, or through some tool you have implemented for the website builder.

**Comment by geoffreymcgill** [comment]
Interesting question. I was expecting the following to work...

#### /parent.md
```md
{{ include "../test/sample-page.md" }}
```

Where `../test/sample-page.md` is in another folder outside of the current project. But, Retype is failing to resolve the path outside of the project. I'll have to figure out why. Maybe we can get this working since the **code snippet** component allows for linking to files outside of the current project, see https://retype.com/components/code-snippet/.

**Reply by beyond9thousand** [reply]
Ah, yes. The code snippet does print the content as a literal code block. Additionally, I was curious if it would be possible to do the same but by calling a markdown file that is hosted online? Like say on another remote repository that I own (or any for that matter I guess)

**Reply by beyond9thousand** [reply]
Since this obviously wouldn't work, as its only a symlink to a **local** repository
![image](https://user-images.githubusercontent.com/77363063/203776973-ce78ba4d-87e0-4abb-be61-071614c7d9a6.png)
